json.go

```diff
@@ -0,0 +1,100 @@
+package clover
+
+import (
+	"os"
+	"encoding/json"
+	"bufio"
+	"io/ioutil"
+	"errors"
+)
+
+// Exporting errors
+var (
+	ErrImportPathUnreachable = errors.New("the import path can't be reached")
+	ErrExportPathExists = errors.New("the export path already exists")
+	ErrExportCollectionEmpty = errors.New("the collection is empty")
+
+)
+
+func pathExists(path string) bool{
+	_, err := os.Stat(path)
+	return err == nil
+}
+
+// Exports a `collection` to JSON format in `filePath`
+func (db *DB) ExportCollection(collectionName string, exportPath string) error{
+	exists, err := db.HasCollection(collectionName)
+	if err != nil{
+		return err
+	}
+	if !exists{
+		return ErrCollectionNotExist
+	}
+	if pathExists(exportPath) {
+    	return ErrExportPathExists
+	}
+
+	count, err := db.Query(collectionName).Count()
+	if err != nil{
+		return err
+	}
+	if count == 0{
```

Comment:
I think that if you export an existing empty collection you should simply get a collection.json file with an empty list.